stdlib/public/core/Sequence.swift

```diff
@@ -959,6 +970,34 @@ extension Sequence {
   }
 }
 
+private enum _StopIteration: ErrorProtocol {
```

Comment:
Sorry, one more thing -- we can't use `private` in the library due to a compiler bug.  Please use `internal`.

Please add a space before the colon to conform to the library coding style.